tests/insert_license_test.py

```diff
@@ -186,6 +187,7 @@ def test_fuzzy_match_license(license_file_path,
         args = ['--license-filepath', license_file_path,
                 '--comment-style', comment_style,
                 '--fuzzy-match-generates-todo',
+                '--fuzzy-ratio-cut-off=75',
```

Comment:
This chang will alter the expected behaviour for all current parameters of thi unit test.
Hence I'd prefer not to change it.
Why do you need to lower it below 85?
Could you instead slitghly change your unit test input maybe?